symbolic link causes NoSuchFileException

after creating a symbolic link(no job is calling symbolic link, seems might be somewhere with verifying files), in Web UI, getting errors below:

> 2017-09-06 12:16:10.169 +1000 [ERROR] (0088@[0:testing]+testing+testing+job1) io.digdag.core.agent.OperatorManager: Task failed with unexpected error: /tmp/digdag-tempdir1542790697699403398/workspace/+testing+testing+job1_1760033870796559295/0.sh
> java.nio.file.NoSuchFileException: /tmp/digdag-tempdir1542790697699403398/workspace/+testing+testing+job1_1760033870796559295/0.sh
> 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
> 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
> 	at sun.nio.fs.UnixFileAttributeViews$Posix.setMode(UnixFileAttributeViews.java:238)
> 	at sun.nio.fs.UnixFileAttributeViews$Posix.setPermissions(UnixFileAttributeViews.java:260)
> 	at java.nio.file.Files.setPosixFilePermissions(Files.java:2045)
> 	at io.digdag.core.archive.ProjectArchives.extractArchive(ProjectArchives.java:82)
> 	at io.digdag.core.archive.ProjectArchives.extractTarArchive(ProjectArchives.java:40)
> 	at io.digdag.core.archive.ProjectArchives.extractTarArchive(ProjectArchives.java:33)
> 	at io.digdag.core.agent.ExtractArchiveWorkspaceManager.withExtractedArchive(ExtractArchiveWorkspaceManager.java:34)
> 	at io.digdag.core.agent.OperatorManager.runWithHeartbeat(OperatorManager.java:135)
> 	at io.digdag.core.agent.OperatorManager.run(OperatorManager.java:119)
> 	at io.digdag.core.agent.MultiThreadAgent.lambda$null$0(MultiThreadAgent.java:127)
> 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
> 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
> 	at java.lang.Thread.run(Thread.java:748)

steps to replicate this issue:
mkdir testing
cd testing
touch 1.sh
ln -s ./1.sh 0.sh

testing.dig
```
_export:
  workflow_name: workflow1

schedule:
  minutes_interval>: 5
  skip_on_overtime: true

+testing:

  +job1:
    sh>: ./1.sh
```

digdag push testing
go to web UI, run it.

digdag download also fails:

> digdag download testing
> 2017-09-06 12:19:25 +1000: Digdag v0.9.13
>   testing/testing.dig
>   testing/0.sh -> 1.sh
> error: /tmp/t2/testing/0.sh (no such file)

--- a/digdag-core/src/main/java/io/digdag/core/archive/ProjectArchives.java
+++ b/digdag-core/src/main/java/io/digdag/core/archive/ProjectArchives.java
@@ -79,7 +79,9 @@ public class ProjectArchives
                     ByteStreams.copy(archive, out);
                 }
             }
-            Files.setPosixFilePermissions(path, getPosixFilePermissions(entry));
+            if (!Files.isSymbolicLink(path)) {
+                Files.setPosixFilePermissions(path, getPosixFilePermissions(entry));
+            }
         }
     }
 

